frontend/src/component/pages/HomePage.tsx

```diff
@@ -356,7 +373,9 @@ export class HomePage extends Component<Props, State> {
           onSearchBarInputChange={this.handleSearchBarInputChange}
           autoCompleteSuggestions={this.state.autoCompleteSuggestions}
           shouldShowSignOutButton={this.state.isUserSignedIn}
+          shouldShowAdminNavButton={this.state.isUserAdmin}
```

Comment:
```suggestion
          shouldShowAdminButton={this.state.isUserAdmin}
```
This is header component. I think it's relatively obvious the button will show up on nav bar.